Library/Formula/fastflow.rb

```diff
@@ -0,0 +1,81 @@
+class Fastflow < Formula
+  desc "C++ parallel programming framework"
+  homepage "http://calvados.di.unipi.it/"
+  url "https://downloads.sourceforge.net/project/mc-fastflow/fastflow-2.0.4.tgz"
+  sha256 "4c5eda03b6aeaabda468bacb2085fdaa481ba2412138303ea0f07ce203de1a3e"
+  head "http://svn.code.sf.net/p/mc-fastflow/code/"
+
+  needs :cxx11
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "zeromq"
+
+  resource "cppzmq" do
+    url "https://github.com/zeromq/cppzmq/raw/05a0256d0eeea8063690fde6a156e14b70ed2280/zmq.hpp"
+    sha256 "bf1c5b38911ca10bfd0826574710eb0c68fbd89b6eaa5e137c34dfbf824c080a"
+  end
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!